binding.cc

```diff
@@ -390,8 +390,7 @@ namespace zmq {
 
   void
   Socket::UV_PollCallback(uv_poll_t* handle, int status, int events) {
-    assert(status == 0);
-
+    if (status != 0) NanThrowError(ExceptionFromError());
```

Comment:
no it wont. 

this is a libuv function as well, so there's no reason to think libzmq would even set an error that `ExceptionFromError()` would pick up there